### ISSUE TITLE
Wrong example shown on forms page

### ISSUE THREAD

**Issue by ajacobs21e** [patternfly/patternfly-org#3357]
https://www.patternfly.org/v4/components/form#validated

The example for "validated" is showing an "invalidated" state. 

(hi!)

**Comment by JustinXHale** [comment]
@ajacobs21e when I click the link you've posted, it shows "Five" (ex1) written into the box which is an invalid number, if five is written numerically "5" (ex2) when get an validation error that states `You must be at least 21 years old to continue
`, if we use "21" it generates a validated green checkmark (ex3). Everything seems to work but maybe we need to add help text to let users know what the validation criteria are? cc: @mmenestr what do you think?

<img width="889" alt="image" src="https://user-images.githubusercontent.com/91554933/213746096-6f3cc47d-8dae-4906-a992-3dfd5e2551c4.png">

<img width="839" alt="image" src="https://user-images.githubusercontent.com/91554933/213746307-c83999ee-0593-4381-96c7-a3ba323b4876.png">

<img width="841" alt="image" src="https://user-images.githubusercontent.com/91554933/213746601-3e66ded1-aca5-44e6-bf9c-3b7b3beed720.png">

**Comment by mmenestr** [comment]
@JustinXHale Allie left Red Hat since this issue was created, but I think maybe we can just make the error text more specific, because it technically _is_ correct.

**Comment by ajacobs21e** [comment]
@mmenestr nope I just made this because I still creep on PF since it's the best design system ever <3

@JustinXHale I see it now that when I interact with the field it does update. Looking at the static page though it was confusing since it defaults to the invalid state and I didn't know I should change the input to see the example of a validated state.

**Comment by mmenestr** [comment]
@ajacobs21e hahahah idk why I read it as "commented last year" instead of "last week". But I agree that titling that example "Validated" is confusing because it makes you think it should show a success state, but I'm also thinking validated might have intended to mean that validation occurred (whether it resulted in an error or success) (?)

@mcarrano @nicolethoen  Do you know what the intention for the "Validated" example at the bottom was? I see that we already have a specific example for "Invalid" so I'm wondering if "Validated" was meant to show success, or if it was just general?

<img width="689" alt="image" src="https://user-images.githubusercontent.com/51165119/214589641-a043567d-c3cf-4b6b-a861-37bb440aad5b.png">

**Comment by mcarrano** [comment]
I agree that the way this example is named is confusing. It would also help to provide some text to explain what this is demonstrating. I remember that when I first saw this I needed to read the code to figure out what was happening. I am going to transfer this to React and tag as a documentation update.
